resources/authgear/templates/en/translation.json

```diff
@@ -1276,6 +1276,7 @@
   "v2.page.settings-mfa-password.default.remove-button-label": "Remove additional password",
   "v2.page.settings-mfa-password.default.title": "Additional password",
   "v2.page.settings-mfa-view-recovery-code.default.regenerate-button-label": "Generate new codes",
+  "v2.page.settings-mfa-view-recovery-code.default.empty": "You do not have any recovery codes yet. Generate new codes by clicking the button below.",
```

Comment:
Rename this key to `v2.page.settings-mfa-view-recovery-code.default.no-existing-codes-description`